core/org/commcare/core/process/XmlFormRecordProcessor.java

```diff
@@ -0,0 +1,44 @@
+package org.commcare.core.process;
+
+import org.commcare.core.interfaces.AbstractUserSandbox;
+import org.commcare.data.xml.DataModelPullParser;
+import org.commcare.data.xml.TransactionParser;
+import org.commcare.data.xml.TransactionParserFactory;
+import org.commcare.xml.CaseXmlParser;
+import org.commcare.xml.LedgerXmlParsers;
+import org.javarosa.core.services.storage.StorageFullException;
+import org.javarosa.xml.util.InvalidStructureException;
+import org.javarosa.xml.util.UnfullfilledRequirementsException;
+import org.kxml2.io.KXmlParser;
+import org.xmlpull.v1.XmlPullParserException;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+/**
+ * Utility methods for processing XML transactions against a user sandbox
+ *
+ * Created by wpride1 on 7/21/15.
+ */
+public class XmlFormRecordProcessor {
```

Comment:
It would be useful to elaborate why this is its own processor that only processes cases and ledgers, and when people should use this v. the other one.